style.css

```diff
@@ -0,0 +1,493 @@
+* { 
+    box-sizing: border-box; 
+    padding: 0;
+    margin: 0;
+    font-family: "Chirp";
+    scroll-behavior: smooth;
+    list-style-type: none;
+    text-decoration: none;
+ }
+/* STYLES */
+ body { 
+    background-color: white;
+ }
+ html {
+    --first-color-light: white;
+    --first-color-medium: #dddddd ;
+    --first-color-dark: ;
+    --second-color-light: #f7f9fa;
+    --second-color-medium: #eceef0;
+    --second-color-dark: #9fa9b4;
+    --third-color-light: #a5d0f5;
+    --third-color-medium: #63a5e1;
+    --third-color-dark: #3779b8;
+    --header-height: 60px;
+}
+.links {
+    color: black;
+    text-decoration: none;
+    transition: background-color .5s;
+}
+.center-wrapper {
+    max-width: 2000px;
+    padding-right: 12px;
+    padding-left: 12px;
+    margin: 0 auto;
+}
+.left-menu-icon {
+    width: 30px;
+}
+.icon {
+    width: 30px;
+}
+.icon-small {
+    width: 25px;
+}
+.icon-x-small {
+    width: 20px;
+}
+.profile-pic-small {
+    width: 50px;
+    border-radius: 50%;
+}
+#logo {
+    margin-left: 10px;
+    width: 40px;
+    margin-top: 10px;
+}
+.profile-pic,
+.profile-pic-someone {
+    width: 70px;
+    height: 70px;
+    border-radius: 50%;
+}
+.java img {
+    display: none;
+}
+.news-icon {
+    max-width: 100px;
+    margin: 20px;
+    border-radius: 10px;
+    width: auto;
+    height: auto;
+}
+h1 {
+}
+h2 {
+    font-size: 20px;
+    font-weight: 900;
+    text-decoration: none;
+    color: black;
+}
+h3 {
+    color: black;
+}
+h4 {
+    color: var(--second-color-dark);
+}
+h5 {
+    color: var(--third-color-medium);
+}
+h6 {
+    font-size: 16px;
+}
+.style-one {
+    color: var(--third-color-medium);
+    font-weight: 900;
+}
+.style-two {
+    color: var(--second-color-dark);
+}
+.style-three {
+    color: var(--second-color-dark);
+    font-size: 20px;
+    font-weight: 540;
+}
+.style-four {
+    margin: 0px 0px 0px 10px;
+    padding: none;
```

Comment:
Using class names such as .style-one, style-two is very practical, I should have done the same.